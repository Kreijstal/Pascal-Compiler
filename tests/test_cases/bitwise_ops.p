program BitwiseOpsTest;
begin
  writeln(8 shl 2);    { 32 }
  writeln(32 shr 2);   { 8 }
  writeln(10 xor 3);   { 9 }
  writeln(12 xor 7);   { 11 }
  writeln(5 shl 1);    { 10 }
  writeln(20 shr 1);   { 10 }
end.
