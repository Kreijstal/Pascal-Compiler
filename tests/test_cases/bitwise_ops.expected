32
8
9
11
10
10
